src/main/java/com/codingnomads/betty/data/batch/batchwriters/FootballMatchRdsWriter.java

```diff
@@ -0,0 +1,33 @@
+package com.codingnomads.betty.data.batch.batchwriters;
+
+import com.codingnomads.betty.data.models.FootballMatchInfo;
+import com.codingnomads.betty.logic.interfaces.FootballMatchesInfoJpaRepository;
+import org.springframework.batch.item.ItemWriter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class FootballMatchRdsWriter implements ItemWriter<List<FootballMatchInfo>> {
+
+    private FootballMatchesInfoJpaRepository matchesInfoJpaRepository;
+
+    @Autowired
+    public FootballMatchRdsWriter(FootballMatchesInfoJpaRepository matchesInfoJpaRepository) {
+        this.matchesInfoJpaRepository = matchesInfoJpaRepository;
+    }
+
+    @Override
+    public void write(List<? extends List<FootballMatchInfo>> items) throws Exception {
+
+        if(items == null){
+            return;
```

Comment:
Hey @alpdurmaz should you be throwing a custom exception here if the list is null? This use of return on a void method is not recommended :)